solutions/4.js

```diff
@@ -0,0 +1,8 @@
+const repeat= (str,num,newStr='',i=0)=>{
+  if(i==num){
+    return newStr;
+  }
+  newStr=newStr+str;
```

Comment:
You can do `newStr+=str`. It is the same as doing `newStr=newStr+str`.

Reply:
thanks!